config.json

```diff
@@ -68,6 +68,16 @@
       ]
     },
     {
+      "uuid": "f9afd650-8103-4373-a284-fa4ecfee7207",
+      "slug": "collatz-conjecture",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "error handling with Result<T, E>"
```

Comment:
We only used the topic `Result` in most[1] other exercises that use `Result`. It seems it would be better to be consistent. But I can't point to any documentation that says "topics must be consistent within a track" (can anyone find any such documentation), so I have no ground to stand on with such a request.

[1]: I say "most" rather than "all" because roman-numerals uses `results`

Reply:
My question about placing an exercise with `result` in this position: In https://github.com/exercism/rust/pull/219 we made an explicit choice to put an exercises that uses `panic` before any exercise that uses `result`:

> I thought it would be nice to get some exposure to `panic` before
introducing `result` in the Hamming exercise.

So now the question is: If this desire is still valid, it should be that `collatz-conjecture` should be placed at least after `grains`.
If the desire is no longer valid for some reason, all we need is to record that decision here, about why it is no longer valid.

Reply:
Until others chime in, I left config.json alone.  I like having this exercise fairly early as it can help students start to look at Result and Option as well.  Of course I will move the exercise if that is the concensus.

Reply:
> Of course I will move the exercise if that is the concensus.

Very well, then my contribution to it will be to suggest placing it between grains and hamming. It seems to require fewer mental switches to go from panic -> result -> result than result -> panic -> back to result.

Reply:
Done!